serial_protocol.md

```diff
@@ -0,0 +1,78 @@
+#Protocol:
+During the first use, or when the chiplet changes environments a “NETWORK” call is expected to initialize the wifi parameters.
+
+Every time a serial connection is established we will expect a “INIT” call after which any subsequent calls can be made, until the connection is closed. 
+
+In the following examples we use $ to represent variables. For example $Host represents your firebase host.
+
+##Response Format Byte
+All responses will be prefixed with one of the following bytes signifying the response type.
+```
+  + If response is ok and a raw string value.
+  * If response is ok and a raw string value prefixed by count of bytes in response then new line.
```

Comment:
Ah, I see so you are more asking if we can merge this with "+", i.e. only return raw strings as "simple strings".

This goes back to our discussions elsewhere of the value of sending a size at all. The big usecases are binary data, which would be transferred as a string and large blobs of text that you may need to prepare storage or other devices for. 

Firebase has a 10mb limit on cells at that size on a MCU you need to do some prep work to handle it.